### PR TITLE
Added support for EIP-868

### PR DESCRIPTION
**Pull request by davemec**
## PR description
Added support for [EIP-868](https://eips.ethereum.org/EIPS/eip-868)

## Fixed Issue(s)
#1695

## Changelog

- [X] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by matkt — COMMENTED** [review]
SGTM

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PingPacketData.java [hunk 1]
```diff
@@ -77,6 +95,13 @@ public void writeTo(final RLPOutput out) {
         .encodeStandalone(out);
     to.encodeStandalone(out);
     out.writeLongScalar(expiration);
+    out.writeBytes(
+        getEnrSeq()
+            .orElseThrow(
+                () ->
+                    new IllegalStateException(
```
Comment (matkt) [inline ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PingPacketData.java:102]: How does Besu behave if this error is thrown ?

Comment (davemec) [reply ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PingPacketData.java:102]: The packet would not be created and an exception is thrown. I did it this way to follow the pattern done for missing from field.

**Review by matkt — APPROVED** [review]
LGTM. just looked at the code

**Pull request merged**